docs/server/multi-cluster.md

```diff
@@ -0,0 +1,392 @@
+| -------- | ------------- | --------------- | ------- |
+```
+
+T = 1: adding event with event ID == 2 & version == 1
+
+View in both Cluster A & B
+
+```
+| -------- | ------------- | --------------- | ------- |
+| Events                   | Version History           |
+| -------- | ------------- | --------------- | ------- |
+| Event ID | Event Version | Event ID        | Version |
+| -------- | ------------- | --------------- | ------- |
+| 1        | 1             | 2               | 1       |
+| 2        | 1             |                 |         |
+| -------- | ------------- | --------------- | ------- |
+```
+
+T = 2: adding event with event ID == 3 & version == 1
+
+View in both Cluster A & B
+
+```
+| -------- | ------------- | --------------- | ------- |
+| Events                   | Version History           |
+| -------- | ------------- | --------------- | ------- |
+| Event ID | Event Version | Event ID        | Version |
+| -------- | ------------- | --------------- | ------- |
+| 1        | 1             | 3               | 1       |
+| 2        | 1             |                 |         |
+| 3        | 1             |                 |         |
+| -------- | ------------- | --------------- | ------- |
+```
+
+T = 3: namespace failover triggered, namespace version is now 2
+adding event with event ID == 4 & version == 2
+
+View in both Cluster A & B
+
+```
+| -------- | ------------- | --------------- | ------- |
+| Events                   | Version History           |
+| -------- | ------------- | --------------- | ------- |
+| Event ID | Event Version | Event ID        | Version |
+| -------- | ------------- | --------------- | ------- |
+| 1        | 1             | 3               | 1       |
+| 2        | 1             | 4               | 2       |
+| 3        | 1             |                 |         |
+| 4        | 2             |                 |         |
+| -------- | ------------- | --------------- | ------- |
+```
+
+T = 4: adding event with event ID == 5 & version == 2
+
+View in both Cluster A & B
+
+```
+| -------- | ------------- | --------------- | ------- |
+| Events                   | Version History           |
+| -------- | ------------- | --------------- | ------- |
+| Event ID | Event Version | Event ID        | Version |
+| -------- | ------------- | --------------- | ------- |
+| 1        | 1             | 3               | 1       |
+| 2        | 1             | 5               | 2       |
+| 3        | 1             |                 |         |
+| 4        | 2             |                 |         |
+| 5        | 2             |                 |         |
+| -------- | ------------- | --------------- | ------- |
+```
+
+</details>
+
+Since Temporal is AP, during failover (change of active Temporal of a namespace), there exist cases where more than one Cluster can modify a workflow, causing divergence of workflow history. Below shows how version history will look like under such conditions.
```

Comment:
```suggestion
Since Temporal is AP, during failover (change of active Temporal Cluster Namespace), there can exist cases where more than one Cluster can modify a Workflow Execution, causing divergence of Workflow  Execution History. Below shows how the version history will look like under such conditions.
```
> Since Temporal is AP, during failover (change of active Temporal of a namespace)

What is AP here?
What is "change of active Temporal of a namespace: